cli/src/main/java/io/specmesh/cli/KafkaProvisionCommand.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2023 SpecMesh Contributors (https://github.com/specmesh)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.specmesh.cli;
+
+import static picocli.CommandLine.Command;
+
+import io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClientConfig;
+import io.specmesh.apiparser.AsyncApiParser;
+import io.specmesh.kafka.KafkaApiSpec;
+import io.specmesh.kafka.Provisioner;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import picocli.CommandLine.Option;
+
+/** SpecMesh Kafka Provisioner */
+@Command(name = "provision", description = "Applied provided specification file to the cluster")
+public class KafkaProvisionCommand implements Runnable {
+
+    @Option(
+            names = {"-br", "--brokerUrl"},
+            description = "Cluster url")
+    private String brokerUrl;
```

Comment:
The standard on Kafka tooling is `--bootstrap-server`, I think.

Also, you need to think about which of these options are _required_. 

For none required, think about default values to avoid nulls.

```suggestion
    @Option(
            names = {"-bs", "--bootstrap-server"},
            required = true,
            description = "Kafka cluster bootstrap url")
    private String brokerUrl = "";
```

Reply:
ok